src/python/WMCore/Services/MSUtils.py

```diff
@@ -0,0 +1,39 @@
+"""
+This module contains a few basic WMCore-related utilitarian functions
+"""
+
+from Utils.CertTools import ckey, cert
+
+from WMCore.Services.pycurl_manager import RequestHandler
+
+
+def makeHttpRequest(srvUrl, payload=None, method='GET', headers=None, encode=True, decode=True, payloadSizeToDump=50):
+    """
+    Fetch documents from given service URL according to the query dictionary
```

Comment:
It does not necessarily fetch a document, does it? Maybe just describe it in a general way (Make an HTTP call to a WM microservice using the underlying pycurl library). Even though it is not limited to MS services, the module name suggests this, so I think it makes sense to write something like that.

In addition, please check the pylint/pycodestyle report from Jenkins and address anything reported in there.

Reply:
ok, done.